flux-refineck/locales/en-US.ftl

```diff
@@ -4,13 +4,24 @@ refineck_goto_error =
     error jumping to join point
 
 refineck_call_error =
-    precondition might not hold
+    precondition might not hold 0
 
 refineck_assign_error =
     assignment might be unsafe
 
 refineck_ret_error =
-    postcondition might not hold
+    postcondition might not hold 0
+
+refineck_condition_span_note =
+    this is the condition that cannot be proved
+
+refineck_call_span_note =
+    inside this specifunction call
+
+refineck_goal_error =
+    {$cond} might not hold
+    .label = {$origin} site
+    .note = a {$cond} cannot be proved at this {$origin} site
```

Comment:
I gave it some thoughts about how to format the errors and I believe something like this would work

```
error[FLUX]: refinement type mismatch
   --> flux-refineck/src/lib.rs:246:9
    |
246 |     assertp(11);
    |     ^^^^^^^^^^^ a precondition cannot be proved
    |
note: this is the precondition that cannot be proved
   --> flux-refineck/src/lib.rs:242:16
    |
242 |     x < 10
    |     ^^^^^^
```

* The `.label` is typically the first thing you read so you should be able to understand the error just by looking at it
* The message at the top can be something generic. I like the idea of it mentioning `refinement types` (or `liquid types`) to denote a generic refinement type error, but I am not convinced about the word mismatch here (that's the word LH uses).
* The `.note` is redundant I'd just remove it